src/MOIWrapper.jl

```diff
@@ -136,30 +170,59 @@ function update_bounds_for_binary_vars!(col_lb::Vector{Float64}, col_ub::Vector{
     end
 end
 
-function update_zeroone_indices(user_optimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
-    ci::Vector{MOI.ConstraintIndex{F,S}}, zero_one_indices::Vector{Int}) where {F, S}
+function update_indices(user_optimizer::MOI.ModelLike, 
+                        mapping::MOIU.IndexMap, 
+                        ci::Vector{MOI.ConstraintIndex{F,S}}, 
+                        list_of_indices::Vector{Int}) where {F, S}
+    
     for i in 1:length(ci)
         f = MOI.get(user_optimizer, MOI.ConstraintFunction(), ci[i])
-        push!(zero_one_indices, mapping.varmap[f.variable].value)
+        push!(list_of_indices, mapping.varmap[f.variable].value)
     end
 end
 
-function update_integer_indices(user_optimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
-    ci::Vector{MOI.ConstraintIndex{F,S}}, integer_indices::Vector{Int}) where {F, S}
-    for i in 1:length(ci)
-        f = MOI.get(user_optimizer, MOI.ConstraintFunction(), ci[i])
-        push!(integer_indices, mapping.varmap[f.variable].value)
+function create_indices_for_bounds(user_optimizer::MOI.ModelLike, mapping::MOIU.IndexMap,
```

Comment:
I don't immediately understand what this function does. A full docstring isn't needed, but a comment would help.